base/docs/basedocs.jl

```diff
@@ -161,7 +161,7 @@ end
     include expressions, literal values, and symbols. For example:
 
         macro sayhello(name)
-            return :( println("Hello, ", \$name) )
+            return :( println("Hello, ", $name) )
```

Comment:
Oh, I see.
I simply copy & paste to the Julia, found it generate an Error.
So should I undo the change on line 164 and fire another pull request?
Excuse me for the first time collaboration.